grakn-kb/src/test/java/ai/grakn/kb/internal/concept/AttributeTypeTest.java

```diff
@@ -173,4 +176,28 @@ public void whenCreatingAResourceTypeOfTypeDate_EnsureTheTimeZoneIsSetTOADefault
             }
         }
     }
+
+    @Test
+    public void whenGettingTheSuperSetViaSupsMethod_ReturnAllOfItsSuperTypes(){
```

Comment:
This test is redundant. We can just stick with the test you wrote `EntityTypeTest#whenGettingTheSuperSetViaSupsMethod_ReturnAllOfItsSuperTypes `

Reply:
I have deleted this test